src/zkevm_specs/bytecode.py

```diff
@@ -162,10 +162,10 @@ def assign_push_table():
 
 
 # Generate keccak table
-def assign_keccak_table(bytecodes: Sequence[bytes], rlc_store: RLCStore):
+def assign_keccak_table(bytecodes: Sequence[bytes], randomness: int):
     keccak_table = []
     for bytecode in bytecodes:
-        hash = rlc_store.to_rlc(keccak256(bytecode), 32)
-        rlc = rlc_store.to_rlc(list(reversed(bytecode)))
+        hash = RLC(bytes(reversed(keccak256(bytecode))), randomness)
+        rlc = RLC(bytes(reversed(bytecode)), randomness, len(bytecode))
```

Comment:
> Which is the purpose of applying the RLC to the keccak inputs? Hasn't it been done previously in `assign_bytecode_circuit`?

This is a mocking `keccak_table`, for lookup the `hash` by `RLC` of `bytecode` and `length`.

> Also, why do we need to reversethe bytecodet?

Because in bytecode circuit, it accumulates the bytes in big-endian order, so we need to reverse it for `RLC`, which takes input as little-endian.

Reply:
Makes complete sense!